app/controllers/staff/extending_organisations_controller.rb

```diff
@@ -0,0 +1,37 @@
+class Staff::ExtendingOrganisationsController < Staff::BaseController
+  def edit
+    @activity = Activity.find(activity_id)
+    authorize @activity
+
+    @delivery_partners = Organisation.delivery_partners
+  end
+
+  def update
+    @activity = Activity.find(activity_id)
+    authorize @activity
```

Comment:
Whilst we're now in a different controller the pundit policy of activity is still applied which is great news.